pkgs/development/python-modules/gasp/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, tkinter, pygame }:
+
+buildPythonPackage rec {
+  pname = "gasp";
+  version = "0.6.1";
```

```suggestion
  version = "0.6.1";

  disabled = pythonOlder "3.6";
```

see https://codeberg.org/GASP/GASP_Code/src/branch/main/setup.cfg#L28